SleepEngine/Engine/Timer.cpp

```diff
@@ -1,19 +1,19 @@
 #include "stdafx.h"
 #include "Timer.h"
-
+#include "GameTemplate/Game.h"
```

Comment:
I guess it is better to use the full path to the file, even if it is in this folder (by the way, it is the rule in our code style on work:)

Reply:
okay